crates/sqlexec/src/planner/physical_plan/create_temp_table.rs

```diff
@@ -100,7 +100,7 @@ async fn create_temp_table(
         && !plan.or_replace
     {
         if plan.if_not_exists {
-            return Ok(RecordBatch::new_empty(Arc::new(Schema::empty())));
+            return Ok(new_operation_batch("create table if not exists"));
```

Comment:
Our expected `create_table` schema didn't align with the original `new_empty` which returned an empty schema. Flightsql was erroring out due to schema mismatch.